python/cuml/fil/fil.pyx

```diff
@@ -181,6 +187,15 @@ cdef class ForestInference_impl():
                             ' to the documentation')
         return algo_dict[algo_str]
 
+    def get_sparse(self, sparse_str):
+        sparse_dict={'AUTO': sparse_t.AUTO,
+                     'DENSE': sparse_t.DENSE,
+                     'SPARSE': sparse_t.SPARSE}
+        if sparse_str not in sparse_dict.keys():
+            raise Exception(' Wrong sparsity selected please refer'
```

Comment:
raise `ValueError` would be typical, also nice to include sparse_str in the error string.

Reply:
Done.